### DISCUSSION TITLE
Snap pointer to a port without a link

### DISCUSSION THREAD

**Discussion by rmnsfx** [clientIO/joint#2612] [Q&A] [answered]
### Introduction

Is it possible to snap a pointer to port without a link? 

When passing near the port (radius) there was a snap and move the pointer to the center of the port area.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment, marked as answer]
No. The position of the pointer (mouse) cannot be controlled using the browser.

Unless you want to translate (or scroll) the paper, so the port moves under the pointer.
